### PR TITLE
Fix build for multiple GOPATHs

### PR DESCRIPTION
**Pull request by anfernee**
The build doesn't work if I have GOPATH like this `/path/a:/path/b`.

### CODE REVIEW

File: hack/install-deps.sh [hunk 1]
```diff
@@ -37,6 +37,15 @@ CNI_DIR=/opt/cni
...
 CONTAINERD_PKG=github.com/containerd/containerd
 
+# Check GOPATH
+if [[ -z "${GOPATH}" ]]; then
+  echo "GOPATH is not set"
+  exit 1
+fi
+
+# For multiple GOPATHs, keep the first one only
+GOPATH=${GOPATH%%:*}}
```
Comment (Random-Liu) [inline hack/install-deps.sh]: Why double `}`? And also the others.

**Comment by Random-Liu** [comment]
@anfernee Thanks for fixing this! Could you sign your commit `git commit -s --amend`.

File: hack/install-deps.sh (hunk 1 above)
Comment (anfernee) [reply hack/install-deps.sh]: my bad, wrong paste.

**Comment by Random-Liu** [comment]
Please sign your commit `git commit -s --amend`.

**Comment by Random-Liu** [comment]
LGTM. Please rebase and ready to merge.

**Comment by Random-Liu** [comment]
@anfernee Thanks for the fix!

**Pull request merged**